playground/storybook/stories/web/Zoom.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Components } from '@chess-tent/web/src/application/types';
+import { User, TYPE_USER } from '@chess-tent/models';
+
+import { withWebNamespace } from '../../utils';
+
+export default {
+  title: 'Components/Zoom',
+} as ComponentMeta<Components['ZoomProvider']>;
+
+const redirectUri =
+  'https://localhost:6006/?path=/story/components-zoom--default';
+
+const user: User = {
```

Comment:
Look at the comment bellow, you can expose user through storybook controls as well.